source/Commands.cpp

```diff
@@ -333,4 +332,59 @@ namespace Commands
   {
     switch_focus_down_or_right(wm::Container::horizontalTiling);
   }
+
+  void switch_workspace(bool next)
```

Comment:
Then how can we know what direction is false or true ? I asked myself the same question but I find it more explicit setting "next" to true or not to know in what direction we are going

Reply:
Maybe `int direction` then? Would avoid the bool to int conversion too. If you're afraid people will pass invalid value, put an assert.

Reply:
Hmm, still prefer this way :laughing:

Reply:
I really don't like next. Sounds like it's a value that's going to be set rather than something controlling the direction of cycling. "forward" could work but sounds wonky because it's a std name I can't think of anything else. Maybe `backwards` and reverse the meaning of the boolean? (seems kinda weird too)